aspnetcore/fundamentals/servers/index.md

```diff
@@ -14,27 +14,57 @@ uid: fundamentals/servers/index
 ---
 # Web server implementations in ASP.NET Core
 
-By [Tom Dykstra](http://github.com/tdykstra), [Steve Smith](http://ardalis.com), [Stephen Halter](https://twitter.com/halter73), and [Chris Ross](https://github.com/Tratcher) 
+By [Tom Dykstra](http://github.com/tdykstra), [Steve Smith](http://ardalis.com), [Stephen Halter](https://twitter.com/halter73), and [Chris Ross](https://github.com/Tratcher)
 
 An ASP.NET Core application runs with an in-process HTTP server implementation. The server implementation listens for HTTP requests and surfaces them to the application as sets of [request features](https://docs.asp.net/en/latest/fundamentals/request-features.html) composed into an `HttpContext`.
 
 ASP.NET Core ships two server implementations:
 
 * [Kestrel](kestrel.md) is a cross-platform HTTP server based on [libuv](https://github.com/libuv/libuv), a cross-platform asynchronous I/O library.
 
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
 * [WebListener](weblistener.md) is a Windows-only HTTP server based on the [Http.Sys kernel driver](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx).
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+* [HttpSys](httpsys.md) is a Windows-only HTTP server based on the [Http.Sys kernel driver](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx).
+
+---
+
 ## Kestrel
 
-Kestrel is the web server that is included by default in ASP.NET Core new-project templates. If your application accepts requests only from an internal network, you can use Kestrel by itself.
+Kestrel is the web server that is included by default in ASP.NET Core new-project templates. 
+
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
 
-![Kestrel to internal network](overview/_static/kestrel-to-internal.png)
+If your application accepts requests only from an internal network, you can use Kestrel by itself.
+
+![Kestrel to internal network](kestrel/_static/kestrel-to-internal.png)
 
 If you expose your application to the Internet, you must use IIS, Nginx, or Apache as a *reverse proxy server*. A reverse proxy server receives HTTP requests from the Internet and forwards them to Kestrel after some preliminary handling, as shown in the following diagram.
 
-![Kestrel to Internet](overview/_static/kestrel-to-internet.png)
+![Kestrel to Internet](kestrel/_static/kestrel-to-internet.png)
+
+The most important reason for using a reverse proxy for edge deployments (exposed to traffic from the Internet) is security. Kestrel is relatively new and does not yet have a full complement of defenses against attacks. This includes but isn't limited to appropriate timeouts, size limits, and concurrent connection limits.
+
+For information about when to use Kestrel with a reverse proxy, see [Kestrel](kestrel.md).
+
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+If your application accepts requests only from an internal network, you can use Kestrel by itself.
+
+![Kestrel to internal network](kestrel/_static/kestrel-to-internal.png)
```

Comment:
Ok ... I'll shoot a few back in a few minutes. The Community Standup snuck up on me (noon CST) and has shifted my :muscle: run back a few hours, so I have 40 minutes now.